docs/source/llms/transformers/task.rst

```diff
@@ -0,0 +1,208 @@
+Tasks in MLflow Transformers Flavor
+===================================
+
+This page provides an overview of how to use the ``task`` parameter in the MLflow Transformers flavor to control the inference interface of the model.
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+In the MLflow Transformers flavor, ``task`` plays a crucial role in determining the input and output format of the model. The ``task`` is a fundamental concept in the Transformers library, which describe the “shape” of each model's API (inputs and outputs) and are used to determine which Inference API and widget we want to display for any given model.
```

Comment:
```suggestion
In the MLflow Transformers flavor, ``task`` plays a crucial role in determining the input and output format of the model. The ``task`` is a fundamental concept in the Transformers library, which describe the “structure” of each model's API (inputs and outputs) and are used to determine which Inference API and widget we want to display for any given model.
```

I'd use `structure` instead of `shape`. `shape` sounds too array-ish to me.

Reply:
I borrowed the term from HF doc, but agree that structure sounds more natural😅